app/src/policy_engine.h

```diff
@@ -151,10 +278,13 @@ namespace scitt
       }
 
       auto phdr_val = protected_header_to_js_val(interpreter, phdr);
+      auto uhdr_val = unprotected_header_to_js_val(interpreter, uhdr);
+      auto payload_val = interpreter.new_array_buffer_copy(payload);
+      auto details_val = verified_details_to_js_val(interpreter, details);
 
       const auto result = interpreter.call_with_rt_options(
         apply_func,
-        {phdr_val},
+        {phdr_val, uhdr_val, payload_val, details_val},
```

Comment:
I suppose this won't break for existing registration policies that only define a single argument for the protected header, right? In other words, the other arguments will be just ignored, correct?

Reply:
Yes it will not affect the existing policies. We do have a couple of configuration tests where arguments vary.